cmd/flags.go

```diff
@@ -78,6 +79,11 @@ func addLocalImageFlag(cmd *cobra.Command) {
 	cmd.Flags().Bool("local", false, "Use an image from local cache")
 }
 
+// addVerifyRegistryFlag add local image flag shared between install, pull-image, upgrade
+func addTLSVerifyFlag(cmd *cobra.Command) {
+	cmd.Flags().Bool("tls-verify", false, "Require HTTPS and verify certificates of registries (default: true)")
```

Comment:
Flag is false by default, not sure if the description is wrong or the actual default, I like it to be on by default or at least that's what I would expect as user.

Reply:
Done